docs/integrations/log_auditbeat_linux.md

```diff
@@ -0,0 +1,327 @@
+uuid: eaa34af4-a194-485c-a33e-4cdd18d064b0
+name: Linux AuditBeat
+
+## Overview
+
+Auditbeat communicates directly with the Linux audit framework, collects the same data as auditd then the data can be stored locally on a JSON format before beeing sent to a log concentrator.
+
+## AuditBeat logs
+
+With AuditBeat it is possible to collect various activities such as the following:
+
+- Authentication: From system module, login dataset retreive User logins, logouts, and system boots
+- File monitoring: From audit module, the file metricset sends events when a file is changed (created, updated, or deleted) on disk. The events contain file metadata and hashes
+- Process command-line parameters: command-lines and arguments are extracted from Linux logs
+- Process monitoring: From system module, process dataset retreive Started and stopped processes
+- Process use of network: From system module, socket dataset retreive Opened and Closed sockets
+
+## Transport to the collector
+
+### Prerequisites
+The following prerequisites are needed in order to setup efficient log concentration:
+
+- Have administrator privileges on the server
+- Traffic towards the Rsyslog must be open on `TCP 514`
+
+### Configure the Client
+
+#### Install and configure AuditBeat
+
+To download and install Auditbeat, use the commands that work with your system:

```

Comment:
We say to use the command that work for your system and then only give the debian version. Maybe we should at list provide a link to th official page

Reply:
This is a documentation for Debian / Linux. The available module and path are not the sames for the others. So it cannot be as detailled and generalised.

Reply:
Fair enough.

I was just thinking that since the name of the intake is `Linux AuditBeat` and not `Debian AuditBeat` we should at least display the installation on RPM systems and I don't think there would have much differences appart from

```
curl -L -O https://artifacts.elastic.co/downloads/beats/auditbeat/auditbeat-7.13.1-x86_64.rpm
sudo rpm -vi auditbeat-7.13.1-x86_64.rpm
```

Reply:
That was a very good point. I changed it. Thank you.